qa/rpc-tests/mergetoaddress_mixednotes.py

```diff
@@ -45,7 +46,11 @@ def run_test(self):
 
         assert_equal(Decimal(self.nodes[1].z_gettotalbalance()["transparent"]), Decimal('0'))
 
-        # Send to t_addr from sprout
+        # Make sure we cannot use "ANY_SPROUT" and "ANY_SAPLING" even if we only have Sprout Notes
```

Comment:
At this point in the test we have both Sprout and Sapling notes (10 ZEC each), I think.

Reply:
You are correct. Fixing now.